### ISSUE TITLE
Only 2 files are displayed in print menu

### ISSUE THREAD

**Issue by anno0** [knutwurst/Marlin-2-0-x-Anycubic-i3-MEGA-S#387]
### Bug Description
Latest Release 1.4.4 of MEGA X Firmware only shows 2 files from SD-Card.
I use Ubuntu Linux. The 2 GB SD-Card is formatted in FAT 32 directly with Linux.

### Steps to Reproduce

1. Copy more than 2 gcode files on the SD-Card
2. Insert the SD-Card
3. Power on the printer
4. Switch to print menu

**Expected behavior:** I would expect that I can navigate through the files and select one of them but I only can see 2 of them.

**Actual behavior:** Only 2 files are displayed.

#### Additional Information
I attached the files from my PC and the printer display.
![info](https://user-images.githubusercontent.com/2607705/200627010-40edfe07-4562-428b-a37b-df3149381969.jpg)
![print](https://user-images.githubusercontent.com/2607705/200627021-59987044-4634-42f2-946f-ad98ee53bc1d.jpg)
![Ubuntu Linux](https://user-images.githubusercontent.com/2607705/200629202-14b0a2db-ff97-417a-bbd7-2f1464f63ecf.png)

**Comment by anno0** [comment]
up

**Comment by knutwurst** [comment]
I'm on it. It should be fixed in the current build.

**Comment by anno0** [comment]
**I think this bug can be closed.** I installed the wrong firmware. As I found out in the wiki, my printer has a DGUS clone display. So I installed the right firmware. Now all files and special menus are displayed.

But now I have another issue. All items on the SD card are displayed with an gcode extension even the trash folder.
![IMG_20221231_095128](https://user-images.githubusercontent.com/2607705/210131174-66e466d7-581f-4093-b620-c694167e76dc.jpg)
